cypress/pom/BonnieFHIR/WI/TestPatientPage.js

```diff
@@ -102,6 +103,20 @@ export const lowerLimitDecimalInputbox = '[data-view-name="lower limit"]'
 export const upperLimitDecimalInputbox = '[data-view-name="upper limit"]'
 export const dataStringInputBox = '[data-view-name="data"]'
 
+// Timing Widget
+export const boundsDropdown = 'select[name="type"]'
```

Comment:
It probably makes sense to use classes in selectors, because there can be multiple inputs with the same name on the page expanded (like units)
 
For bounds:  '.timing-repeat-bounds-view select[name="type"]'
duration units: '.timing-repeat-duration-unit-view input[name="unit"]'
count input '.timing-repeat-count-view input[data-cy=positive-integer-input]'

Time repeat classes:

timing-repeat-bounds-view
timing-repeat-count-view
timing-repeat-count-max-view
timing-repeat-duration-view
timing-repeat-duration-max-view
timing-repeat-duration-unit-view
timing-repeat-frequency-view
timing-repeat-frequency-max-view
timing-repeat-period-view
timing-repeat-period-max-view
timing-repeat-period-unit-view
period-view timing-repeat-dayofweek-view
timing-repeat-timeofday-view
timing-repeat-when-view
timing-repeat-offset-view

Reply:
Il double check those, I did add them to most of them, I think I had trouble finding the parent class in a few elements.

Reply:
I'll double check those and add those class names